src/services/item/service.ts

```diff
@@ -430,7 +431,13 @@ export class ItemService {
   }
 
   /////// -------- COPY
-  async copy(actor: Actor, repositories: Repositories, itemId: UUID, args: { parentId?: UUID }) {
+  async copy(
+    actor: Actor,
+    repositories: Repositories,
+    itemId: UUID,
+    args: { parentId?: UUID },
+    name?: string,
```

Comment:
If possible I would keep the newName in the `args` argument kind of like a optional argument instead of adding a new parameter to the function.
```suggestion
    args: { parentId?: UUID; newName?: string },
```